lib/core/src/Data/Quantity.hs

```diff
@@ -102,6 +104,11 @@ instance (KnownSymbol unit, FromJSON a) => FromJSON (Quantity unit a) where
           where
             u = symbolVal proxy
 
+instance FromText b => FromText (Quantity sym b) where
+    fromText = fmap Quantity . fromText
+
+instance ToText b => ToText (Quantity sym b) where
+    toText (Quantity b) = toText b
```

Comment:
:+1:

Reply:
alternetively we might also use `sym` here and have something like "20lovelace" "20ada" to parse differently and to do conversion automatically. Will put it on the blackboard